lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -144,7 +146,7 @@ def get(self, request, *args, **kwargs):
 
         # Get has_scheduled_content data
         transformers = BlockStructureTransformers()
-        transformers += [start_date.StartDateTransformer()]
+        transformers += [start_date.StartDateTransformer(), ContentTypeGateTransformer()]
```

Comment:
This transformers list is starting to look a little bit like the list in `get_course_block_access_transformers` (though that list still has five extra transformers). I'm assuming it was an intentional choice not to use that list? (Which includes things like VisibilityTransformer, so I'm assuming we wanted to show non-visible content in this grades list, like unreleased exams?)

Reply:
I'm not sure if this was intentional, the transformers weren't a part of this API until this PR: https://github.com/edx/edx-platform/pull/27323/files

@MatthewPiatetsky do you know?